### PR TITLE
DestroyOutsideBehavior JavaScript function annotations

### PR DESCRIPTION
**Pull request by Wend1go**

### CODE REVIEW

**Comment by blurymind** [comment]
So does this destroy an object that leaves the viewport? I haven't actually used it before.
Wonder if there is a way to detect if a game object is inside/outside the viewport

File: Extensions/DestroyOutsideBehavior/destroyoutsideruntimebehavior.js [hunk 1]
```diff
@@ -38,10 +38,18 @@ gdjs.DestroyOutsideRuntimeBehavior.prototype.doStepPostEvents = function(runtime
     }
 };
 
+/**
+ * Set an additional border to the camera viewport as a buffer before the object gets destroyed.
+ * @param {number} val Border in pixel.
```
Comment (4ian) [inline Extensions/DestroyOutsideBehavior/destroyoutsideruntimebehavior.js]: in pixels

File: Extensions/DestroyOutsideBehavior/destroyoutsideruntimebehavior.js [hunk 2]
```diff
@@ -38,10 +38,18 @@ gdjs.DestroyOutsideRuntimeBehavior.prototype.doStepPostEvents = function(runtime
...
+ */
 gdjs.DestroyOutsideRuntimeBehavior.prototype.setExtraBorder = function(val) {
     this._extraBorder = val;
 };
 
+/**
+ * Get the additional border of the camera viewport buffer which triggers the destruction of an object.
+ * @return {number} The addidional border around the camera viewport in pixel
```
Comment (4ian) [inline Extensions/DestroyOutsideBehavior/destroyoutsideruntimebehavior.js]: typo: "additional" and ", in pixels."

**Comment by 4ian** [comment]
Thanks for this, add comments about typos :)

@blurymind Yes: http://wiki.compilgames.net/doku.php/gdevelop5/behaviors/destroyoutside

> Wonder if there is a way to detect if a game object is inside/outside the viewport

There is no condition for this now. I recall @Lizard-13 talking about this too. With events it can be done simply by checking the position of the object against Camera X/Y and using Camera size, but it's a bit cumbersome. A condition could help make this easier.

The behavior is still the way to go in case you want to simply have your instance destroy when it's out of the viewport.

**Comment by 4ian** [comment]
Sweet 👍

**Pull request merged**